src/Form/AddChildrenWizard/ChildBatchProcessor.php

```diff
@@ -36,6 +37,9 @@ protected function getNode($info, array $values) : NodeInterface {
       throw new \Exception("Failed to create node.");
     }
 
+    // Assign the new node to the same group as parent.
+    ParentGroupInheritanceHelper::addEntityToParentGroup($node);
```

Comment:
We probably don't want to do this directly in the class in `islandora`, especially without creating a dependency on the module containing the other code.

Could possibly dispatch a specific event or something of the like? Subscribe to such event over in `enhanced_collection_pages` or `dgi_group` or whatever? Could be convenient to implement a separate `::addToSameGroup($node, $parent)` method ([which could be refactored out of the current method](https://github.com/discoverygarden/enhanced_collection_pages/blob/7f7701bfc6c03299a4668baaa6474c35f6de0561/src/Form/ParentGroupInheritanceHelper.php#L67-L92)), as we already have the `$parent` loaded here, and so could skip about the [first half of the function](https://github.com/discoverygarden/enhanced_collection_pages/blob/7f7701bfc6c03299a4668baaa6474c35f6de0561/src/Form/ParentGroupInheritanceHelper.php#L45-L66).

Could see such generic event dispatch going into the module upstream.